maquetado/scss/navbar.scss

```diff
@@ -19,5 +19,5 @@
 }
 
 .navbar p {
```

Comment:
dont use tags in scss

Reply:
Not using tags in previous version (PR Fixes in navbar pr)